lib/web_ui/dev/roll_fallback_fonts.dart

```diff
@@ -65,6 +65,13 @@ class RollFallbackFontsCommand extends Command<bool>
       ...await _processSplitFallbackFonts(client, splitFallbackFonts),
       ...await _processFallbackFonts(client, apiFallbackFonts),
     ];
+
+    final String? failedUrl = await _checkForLicenseAttributions(client, fallbackFontInfo);
```

Comment:
Maybe return a `List<String>` or `Set<String>` with all the fonts that might have failed, and then do `if (failedUrls.isNotEmpty)` to throw? That way we can report all the fonts that have a problem at once.

Reply:
Great idea!